blog/_posts/2021-05-26-wip-vs-harness.md

```diff
@@ -0,0 +1,138 @@
+---
+slug: "wip-vs-harness"
+date: 2021-05-26
+author: nick
+layout: blog
+title: "WIP Development vs Harness Development"
+subtitle: "Local K8s dev, GitOps, and when to use them together"
+description: "Local K8s dev, GitOps, and when to use them together"
+image: "/assets/images/wip-vs-harness/cover.jpg"
+image_caption: "<i>Everybody Rides in the Whip at Midland Beach</i>. Via <a href='https://digitalcollections.nypl.org/items/510d47d9-c024-a3d9-e040-e00a18064a99'>The New York Public Library</a>."
+tags:
+  - gitops
+  - localdev
+---
+
+In the #tilt chat last week, `renaudguerin` asked:
+
+> I’m the new DevOps guy in a small startup, and migrating us away from broken
```

Comment:
I would replace "away from broken in-house tooling" with an ellipsis, tbqh. Don't wanna be broadcasting someone criticize their own company.

Reply:
done